railcontrol.conf.dist

```diff
@@ -8,6 +8,10 @@ dbfilename = railcontrol.sqlite
 # Default dbkeepbackups is 10
 dbkeepbackups = 10
 
+# If this is set to :: the webserver will listen to all addresses
```

Comment:
Instead of using :: and ::1 I would prefer using "any" and "localhost". The meaning of these words is known to more people then the special IPv6 numbers.

Reply:
changed it to `any` / `localhost`.